.github/pull_request_template.md

```diff
@@ -1,29 +1,30 @@
 ## Proposed changes
 
-Describe the big picture of your changes here to communicate to the maintainers why the pull request is necessary and why we should accept it. If it fixes a previously reported bug, be sure to link to that issue. Please also summarize the changes to the code.
+Contributions to PorePy are highly appreciated. Clearly explain to the maintainers why this pull request (PR) is needed and why it should be accepted. If this PR is solving an issue, explain how it is done. Please, also summarise the changes to the code. 
 
 ## Types of changes
 
-What types of changes does your code introduce to PorePy?
+What types of changes does this PR introduce to PorePy?
 _Put an `x` in the boxes that apply_
 
+- [ ] Minor change (e.g., dependency bumps, broken links, etc.)
 - [ ] Bugfix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-- [ ] Testing (contribution related mainly to testing of existing functionality)
-- [ ] Documentation update (if none of the other choices apply)
+- [ ] Testing (contribution related to testing of existing or new functionality)
+- [ ] Documentation (contribution related to adding, improving, or fixing documentation)
+- [ ] GitHub actions workflows (e.g., workflow files `*.yml`)
+- [ ] Back-end working principle (e.g., internal logic, data storage, data manipulation, integration, etc.)
 - [ ] Other: 
 
 ## Checklist
 
-_Enter one &check; (`&check;`) in each row. The table can be updated after creating the PR._
+_Put an `x` in the boxes that apply or explain briefly why the box is not relevant._
 
-||Yes|No|NA|
-|---|---|---|---|
-|The update is covered by the  test suite |_|_|_|
-|The documentation is up-to-date |_|_|_|
-|I have not duplicated existing functionality |_|_|_|
+- [ ] The documentation is up-to-date.	
+- [ ] Static typing is included in the update.
+- [ ] This PR does not duplicated existing functionality.
+- [ ] The update is covered by the test suite (including new tests added in the PR).
+- [ ] The added tests consider edge/limiting cases.
```

Comment:
I think this is not always relevant. And when it is, it should be covered by the previous item. I suggest (but don't insist) to remove it.

Reply:
I agree. At the end of the day, I will not use it at all. @pmgbergen/porepy-maint what is your take on this?

Reply:
Fine with me.